sycl/include/sycl/ext/xilinx/fpga/memory_properties.hpp

```diff
@@ -14,6 +14,10 @@
 
 __SYCL_INLINE_NAMESPACE(cl) {
 
+// TODO(lforg37): [Technical Debt]: Merge properties for hbm_bank and ddr_bank with a template type for the 
+//                                  type of memory bank. 
```

Comment:
Other technical debt: line too long and trailing spaces. Or is this to have this message not being forgotten since it will ring the bell every time we run clangd/clang-tidy/clang-format ? :-)